main.py

```diff
@@ -1,22 +1,40 @@
 import os
+import time
 import json
 from timeit import default_timer as timer
 from pymongo import MongoClient
+from elasticsearch import Elasticsearch
 
 from fhirstore import FHIRStore
 
+
 if __name__ == '__main__':
-    client = MongoClient()
-    store = FHIRStore(client, "fhirstore")
+    client = MongoClient(username="arkhn", password="SuperSecurePassword2019")
+
+    client_es = Elasticsearch(
+            ['http://localhost:9200'],
+            http_auth=("elastic", "SuperSecurePassword2019")
+    )
+    # uncomment the next 2 following line if you wish to activate
+    # the replication mode of mongo (required by monstache)
+
+    # res = client.admin.command("replSetInitiate", None)
+    # time.sleep(1)
```

Comment:
From the comment here, and since we have now monstache by default in our docker compose, should we uncomment?

Reply:
This file is only used for testing the code locally, it is not used by docker or anything else.